api/src/main/resources/swagger3codegen/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -406,14 +406,15 @@ public class ApiClient {
      */
     public ApiClient setDebugging(boolean debugging) {
         if (debugging != this.debugging) {
+            var builder = httpClient.newBuilder();
```

Comment:
This is the only mustache template with significant changes (not just renames and signature updates)

This builder is necessary because `httpClient` has become immutable, and therefore we can't modify the interceptors in-place.  (Keeping the old code as-is results in an unmodifiable exception.)

Reply:
I was able to confirm that this behavior worked as expected by debug-stepping through the test that failed with the unmodifiable exception before this change.